lib/libutee/include/apdu_pta.h

```diff
@@ -0,0 +1,30 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/*
+ * Copyright (C) 2021, Foundries Limited
+ */
+
+#ifndef __APDU_PTA_H
+#define __APDU_PTA_H
+
+#define PTA_APDU_UUID { 0x3f3eb880, 0x3639, 0x11ec, \
+			{ 0x9b, 0x9d, 0x0f, 0x3f, 0xc9, 0x46, 0x8f, 0x50 } }
+/*
+ * ADPU based communication with the Secure Element
+ *
+ * [in]  value[0].a           APDU request type.
```

Comment:
You mean explicit values for the IDs carries in this type parameter.

Reply:
yes, once we agree on the libseteec names, I'll just import them here